src/core/dynamic_batch_scheduler.cc

```diff
@@ -122,6 +122,66 @@ DynamicBatchScheduler::Create(
   return Status::Success;
 }
 
+Status
+DynamicBatchScheduler::Create(
+    const uint32_t runner_id_start, const uint32_t runner_cnt, const int nice,
+    const StandardInitFunc& OnInit, const StandardWarmupFunc& OnWarmup,
+    const StandardRunFunc& OnSchedule,
+    const StandardShapeTensorPeekFunc& OnPeek,
+    const bool dynamic_batching_enabled,
+    const std::unordered_map<std::string, bool>& enforce_equal_shape_tensors,
+    const bool preserve_ordering,
+    const std::set<int32_t>& preferred_batch_sizes,
+    const uint64_t max_queue_delay_microseconds,
+    const ModelQueuePolicy& default_queue_policy,
+    const uint32_t priority_levels, const ModelQueuePolicyMap& queue_policy_map,
+    std::unique_ptr<Scheduler>* scheduler)
+{
+  DynamicBatchScheduler* dyna_sched = new DynamicBatchScheduler(
+      runner_id_start, runner_cnt, OnInit, OnWarmup, OnSchedule, OnPeek,
+      dynamic_batching_enabled, enforce_equal_shape_tensors, preserve_ordering,
+      preferred_batch_sizes, max_queue_delay_microseconds);
+  std::unique_ptr<DynamicBatchScheduler> sched(dyna_sched);
+
+  sched->queue_ =
```

Comment:
Don't you need to define queue_ in the other Create function? Also, there is a lot of duplicate code here. Can you instead just have a single Create that has default values for the new fields and then when priority_levels == 0 just create queues_ appropriately.